ERP-BACKEND/app/routers/crm.py

```
@@ -364,6 +404,231 @@ def delete_deal(deal_id: int, db: Session = Depends(get_db), current_user = Depe
+    if entity_type:
+        query = query.filter(FollowUp.entity_type == entity_type)
+    if entity_id is not None:
+        query = query.filter(FollowUp.entity_id == entity_id)
+    if completed is not None:
+        query = query.filter(FollowUp.completed == completed)
+    if priority:
+        query = query.filter(FollowUp.priority == priority)
+    if upcoming:
+        from datetime import datetime, timezone
+        now = datetime.now(timezone.utc)
+        query = query.filter(FollowUp.completed == False, FollowUp.scheduled_for >= now)
```

Ruff (0.16.1)

[error] Avoid equality comparisons to `False`; use `not FollowUp.completed:` for false checks

Replace with `not FollowUp.completed`

(E712)